[2024-01-16T17:22:03.497+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: branching_with_variables.determine_branch manual__2024-01-16T15:21:54.370881+00:00 [queued]>
[2024-01-16T17:22:03.502+0200] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: branching_with_variables.determine_branch manual__2024-01-16T15:21:54.370881+00:00 [queued]>
[2024-01-16T17:22:03.502+0200] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-01-16T17:22:03.522+0200] {taskinstance.py:2192} INFO - Executing <Task(BranchPythonOperator): determine_branch> on 2024-01-16 15:21:54.370881+00:00
[2024-01-16T17:22:03.526+0200] {standard_task_runner.py:60} INFO - Started process 55866 to run task
[2024-01-16T17:22:03.529+0200] {standard_task_runner.py:87} INFO - Running: ['airflow', 'tasks', 'run', 'branching_with_variables', 'determine_branch', 'manual__2024-01-16T15:21:54.370881+00:00', '--job-id', '141', '--raw', '--subdir', 'DAGS_FOLDER/branching_w_variables.py', '--cfg-path', '/tmp/tmp53m79pd2']
[2024-01-16T17:22:03.530+0200] {standard_task_runner.py:88} INFO - Job 141: Subtask determine_branch
[2024-01-16T17:22:03.573+0200] {task_command.py:423} INFO - Running <TaskInstance: branching_with_variables.determine_branch manual__2024-01-16T15:21:54.370881+00:00 [running]> on host DESKTOP-9NHC01K.
[2024-01-16T17:22:03.633+0200] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='branching_with_variables' AIRFLOW_CTX_TASK_ID='determine_branch' AIRFLOW_CTX_EXECUTION_DATE='2024-01-16T15:21:54.370881+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-01-16T15:21:54.370881+00:00'
[2024-01-16T17:22:03.639+0200] {python.py:201} INFO - Done. Returned value was: filter_by_southeast
[2024-01-16T17:22:03.639+0200] {branch.py:35} INFO - Branch into filter_by_southeast
[2024-01-16T17:22:03.639+0200] {skipmixin.py:178} INFO - Following branch filter_by_southeast
[2024-01-16T17:22:03.643+0200] {taskinstance.py:2699} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/models/taskinstance.py", line 433, in _execute_task
    result = execute_callable(context=context, **execute_callable_kwargs)
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/operators/python.py", line 233, in execute
    return self.do_branch(context, super().execute(context))
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/operators/branch.py", line 36, in do_branch
    self.skip_all_except(context["ti"], branches_to_execute)
  File "/home/mahmoud/airflow_env/lib/python3.10/site-packages/airflow/models/skipmixin.py", line 213, in skip_all_except
    raise AirflowException(
airflow.exceptions.AirflowException: 'branch_task_ids' must contain only valid task_ids. Invalid tasks found: {'filter_by_southeast'}.
[2024-01-16T17:22:03.645+0200] {taskinstance.py:1138} INFO - Marking task as FAILED. dag_id=branching_with_variables, task_id=determine_branch, execution_date=20240116T152154, start_date=20240116T152203, end_date=20240116T152203
[2024-01-16T17:22:03.661+0200] {standard_task_runner.py:107} ERROR - Failed to execute job 141 for task determine_branch ('branch_task_ids' must contain only valid task_ids. Invalid tasks found: {'filter_by_southeast'}.; 55866)
[2024-01-16T17:22:03.703+0200] {local_task_job_runner.py:234} INFO - Task exited with return code 1
[2024-01-16T17:22:03.763+0200] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
